docs: add link to secret key rotation script (#12072)
langflow-ai/langflow: docs/docs/Develop/api-keys-and-authentication.mdx

          - LANGFLOW_SECRET_KEY=${LANGFLOW_SECRET_KEY}
        ```

> #### Rotating the secret key {#rotating-the-secret-key}

Reviewer: ```suggestion
#### Rotate the secret key {#rotating-the-secret-key}
```

> You should rotate `LANGFLOW_SECRET_KEY` if the key may have been compromised or as part of routine security practice.

Reviewer: ```suggestion
Rotate `LANGFLOW_SECRET_KEY` if the key might have been compromised and as part of your routine credential management practices.
```
